pkg/logger/logger.go

```diff
@@ -77,9 +77,9 @@ func (l Level) AsSevereAs(log Level) bool {
 
 var (
 	NoneLvl    = Level{id: 0, severity: 0}
-	DebugLvl   = Level{id: 3, severity: 100}
-	VerboseLvl = Level{id: 2, severity: 200}
 	InfoLvl    = Level{id: 1, severity: 300}
+	VerboseLvl = Level{id: 2, severity: 200}
```

Comment:
haha, i think there's an edit war where @maiamcc wanted these to go in severity order

Reply:
😬 — I actually vaguely knew about that. I was playing with this and hadn't actually meant to commit it. I withdraw my end of this tug of war.